unbxdAutosuggest.js

```diff
@@ -736,8 +741,10 @@ var unbxdAutoSuggestFunction = function ($, Handlebars, params) {
 				}
 			});
 
-			if (typeof this.options.onItemSelect == "function" && data.type !== "POPULAR_PRODUCTS_FILTERED") {
-				this.options.onItemSelect.call(this, data, this.currentResults[data.type][parseInt(data['index'])]._original, e);
+			if (typeof this.options.onItemSelect == "function" && data.type !== "POPULAR_PRODUCTS_FILTERED" && data.type !== "POPULAR_PRODUCTS") {
+				this.options.onItemSelect.call(this, data, this.currentResults['COMBINE_RESULTS'][parseInt(data['index'])]._original, e);
```

Comment:
added sorting functionality as well, there are customers who look for sorting irrespective of suggestions since, we are making changes in library all things can be tested once so that next time we no need to change library simply we can handle from config

Reply:
Sorting is added if infields are not present in suggestions, let's enhance on that?
Line 1597